src/inversion_scripts/run_inversion.sh

```diff
@@ -3,6 +3,9 @@
 #SBATCH -N 1
 #SBATCH -o run_inversion_%j.out
 #SBATCH -e run_inversion_%j.err
+#SBATCH -t 0-6:00
+#SBATCH --mem=20000
+#SBATCH -p seas_compute,shared,huce_intel
```

Comment:
remove these defaults, this will override the specified values in the IMI config file

Reply:
Agreed. I think this was added while debugging, but should be removed.

Reply:
Now removed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/c53f4d519860a678a5dec31f403770c497cfab43.